examples/firebase-auth-firestore/package.json

```diff
@@ -25,6 +28,6 @@
     "typescript": "^4.6.3"
   },
   "engines": {
-    "node": ">=14"
+    "node": "14"
```

Comment:
```suggestion
    "node": ">=14"
```

Reply:
This was previously discussed here https://github.com/remix-run/remix/pull/3398#discussion_r890170777

Reply:
In fact as per my previous comment at https://github.com/remix-run/remix/pull/3398#discussion_r896676765 I think this should be set to 16